spec/integration/network/http_pool_spec.rb

```diff
@@ -72,8 +72,6 @@ def connection(host, port)
           expect {
             http.get('/')
           }.to raise_error { |err|
-            pending("PUP-8213") if RUBY_VERSION.to_f >= 2.4
-
             expect(err).to be_instance_of(Puppet::SSL::CertMismatchError)
             expect(err.message).to match(/\AServer hostname '#{wrong_hostname}' did not match server certificate; expected one of (.+)/)
```

Comment:
I feel like this error message should be more like:
Configured Server hostname #{wrong_hostname} did not match any name in server certificate: (.+).

Reply:
The term `Configured` is a bit ambiguous. Is the client, server or server cert misconfigured? Also it sounds like it may be referring to the `server` setting, but we don't actually know how the agent resolved the server's fqdn. It could have been `server`, `server_list`, or SRV records. About the "any name" part, I think that's already handled in the "expected one of ..." part.